### DISCUSSION TITLE
Does Sumatrapdf support PDF/A-3 format?

### DISCUSSION THREAD

**Discussion by Yukisama** [sumatrapdfreader/sumatrapdf#4207] [Documentation]
Does Sumatrapdf support PDF/A-3 format?

I'm attempting to open a PDF document using SumatraPDF, but I'm encountering an error. 

Upon further investigation, I've discovered that the file format is PDF/A-3b. 

I'm wondering if this format is compatible with SumatraPDF.

Could anyone please confirm whether SumatraPDF supports PDF/A-3 documents?

OS: Windows10 pro x64
Version: Sumatrapdf 3.6.16056

**Comment by GitHubRulesOK** [comment]
PDF/A-3 is simply a subset of PDF so should be compatible
You need to provide an example, but beware PDF/A as a label does not mean a PDF is automatically usable (which it should like PFD/A-1) since it is an XMP name tag for a type of PDF with extended features!

PDF/A = compliant with standards (! not always the case)
3= Use of ISO 32000-1 (1.7) with support for **embedded files** such as invoices
B= NON compliant for use with user Accessible Applications
Level B conformance requires only that standards necessary for the reliable reproduction of a document's visual appearance be followed

Even more specialist would be PDF/UA for assistive readers

The problem with PDF/A-3 it is a SUPERSET of type -2 thus all very exotic types of PDF are possible
specialist JPEG 2000 image compression.
transparency effects and **layers**.
embedding of OpenType fonts.
provisions for digital signatures in accordance with PAdES standard.
embedding PDF/A files to facilitate archiving of sets of documents with a single file.


Thus PDF/A-2 onwards are not traditional PDF printing /reading structures

**Comment by kjk** [comment]
There is a number of reasons we might fail to open a file and that's all I can say without having the file that fails to open.

If you want me to look at it, open a bug report and provide the file that fails to open: https://github.com/sumatrapdfreader/sumatrapdf/issues/new/choose
